### DISCUSSION TITLE
Configuring must_draw_borders

### DISCUSSION THREAD

**Discussion by andmis** [kovidgoyal/kitty#4208] [Q&A] [answered]
In `kitty/layout/base.py`, there is the following line:

```
    must_draw_borders = False  # can be overridden to customize behavior from kittens
```

What I would like is for this to be `True` for _every_ window I use -- I want the border always to be drawn, including in OS windows with only a single pane/terminal.

So far, I have tried using watchers, kittens, and editing the source code (changing `False` to `True` in that line of code, in a local clone of the kitty GitHub repo) and using `KITTY_DEVELOP_FROM`. I couldn't get watchers or kittens to work. Editing the source worked, except after a macOS upgrade suddenly kitty stopped launching until I turned off `KITTY_DEVELOP_FROM`. (I confirmed that the installed kitty version was the same as the commit checked-out in my clone of the repo.) So basically I think I have tried pretty hard to achieve this goal. I can achieve what I want by editing the source code, but I would prefer a more robust solution.

Any advice?

**Comment by kovidgoyal** [comment, marked as answer]
What's preventing you from getting watchers to work?
If you run kitty from master or nightly you can create "global" watchers
that run on every window, and those should do the trick pretty easily.

**Comment by andmis** [comment]
Thanks for the suggestion. I am not on the master/nightly builds -- I think I'll wait until that feature hits the main release version and try again.
